app/Http/Controllers/Entity/InventoryController.php

```diff
@@ -156,6 +155,28 @@ public function update(UpdateInventory $request, Campaign $campaign, Entity $ent
             ]));
     }
 
+    /**
+     */
+    public function deleteSection(Campaign $campaign, Entity $entity, Request $request)
+    {
+        $this->authorize('update', $entity->child);
+
+        $position = $request->get('position');
+
+        if (isset($position) && $position !==  __('entities/inventories.default_position')) {
```

Comment:
This smells like a bad idea, sending strings for db lookups in the database. Send an entity_inventory object instead and directly use the ->position from it, much safer